File: Java/gesbrasilburger/src/main/java/sn/brasilburger/entity/Paiement.java
package sn.brasilburger.entity;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString

public class Paiement {
    private Integer id;
    private LocalDateTime datePaiement;
    private Double montant;
    private ModePaiement mode;
    private Commande commande;
}

File: CSharp/brasilBurger/Models/Paiement.cs
namespace brasilBurger.Models
{
    public enum ModePaiement
    {
        OM,
        WAVE
    }
    public class Paiement
    {
        public int Id { get; set; }
        public DateTime DatePaiement { get; set; }
        public decimal Montant { get; set; }
        public ModePaiement Mode { get; set; }
        public int CommandeId { get; set; }
        public Commande Commande { get; set; }
    }
}